pkg/kf/commands/utils/command_override_fetcher_test.go

```diff
@@ -411,6 +412,12 @@ func TestCommandOverrideFetcher(t *testing.T) {
 
 			f := utils.NewCommandOverrideFetcher(kfFake, fakeBuild, tc.BuildTailer(t), tc.BuildImage(t), &config.KfParams{Namespace: tc.Namespace})
 			results, err := f.FetchCommandOverrides()
+
+			// Default to silencing the error output.
+			for _, cmd := range results {
+				cmd.SetOutput(&bytes.Buffer{})
```

Comment:
This seems like it slipped into this commit, I don't think we should be overriding the output except in the case of tests.

Reply:
This is more of a byproduct of #60. The test was outputting the usage string when the test was asserting on an error. I'll remove it and we can solve this when fixing #60